src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -1,16 +1,15 @@
 package edu.berkeley.ce.rockslicing
 
-import breeze.linalg.DenseVector
-
 import scala.io.Source
 import scala.util.Try
 
 object InputProcessor {
   // Processes input file: Add rock volume faces and joints to respective input list
-  def readInput(inputSource: Source): Option[(Array[Double],
-                                             Seq[Face], Seq[Joint])] = {
+  def readInput(inputSource: Source): Option[ (Array[Double], Array[Double],
+    Seq[Array[Double]], Seq[JointSet]) ] = {
```

Comment:
I'm leaning towards us using more abstraction for the rock volume as well. Is there any reason we can't just use our `Face` class, and return a `Seq` of `Face` instances? If I'm not mistaken, it should already have all of the necessary fields, right?